### ISSUE TITLE
Comment markdown preview

### ISSUE THREAD

**Issue by bill-mybiz** [ibgib/ibgib#168]
When typing in the comment section, if the screen is big enough, it should show a markdown preview. Or perhaps, should be a tabbed interface.

**Comment by OvermindDL1** [comment]
I implemented both at work, it is a side-by-side preview, like the forums, but you can disable the preview or the input box by clicking a button if you want to see it full size, it seems to work well.  :-)

**Comment by bill-mybiz** [comment]
Ah, cool! :sunglasses: 

Is it an app that is mobile-friendly also? Do you handle checking for the screen size? I have found that the elixir forum implementation is a tad unwieldy when using my phone. I usually end up just using "Request desktop site" and that's not really an option with ibGib... _(I found this out when I was clicking things and all of the sudden my pop up menu was tiny with these tiny svg shapes...it took me 15 minutes to figure out it had nothing to do with my code! :roll_eyes: :laughing:)_

**Comment by OvermindDL1** [comment]
> Is it an app that is mobile-friendly also? Do you handle checking for the screen size?

Yep, just moved from being side by side to being vertical and the preview is by default disabled.  Still playing with the design overall, but not as bad on mobile as discourse at least.  :-)
